1.0/docs/start/first-element/step-2.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 2: Add Local DOM"
+subtitle: Build your first Polymer element
+---
+
+<style>
+.caption + pre,.caption + .prettyprint {
+  margin-top: 0px;
+}
+</style>
+
+Next, you'll create a simple element that displays an icon.
+
+In this step, you'll learn about:
+
+*   Creating a custom element using Polymer.
+*   Working with local DOM.
+
+
+<div class="alert alert-info"><strong>Learn more: local DOM.</strong> Local DOM
+lets you add a <em>scoped</em> DOM tree inside an element, with local styles and
+markup that are decoupled from the rest of the web page. The element containing
+the local DOM tree is called a <em>host element</em>. To learn more, see <a
+href="https://www.polymer-project.org/1.0/docs/devguide/local-dom.html">Local
+DOM</a> in the Polymer library docs.
+</div>
+
+## Edit icon-toggle.html
+
+Click on `icon-toggle.html `in the root of the Chrome Dev Editor
+navigation panel. This file contains the skeleton of a custom element.
+
+Unlike most HTML files, this file <em>won't display anything if you load it in a
+browser</em>—it just <em>defines</em> a new element. The demo <em>imports</em>
```

Comment:
the emphasis around imports started to feel like I was being talked down to. I'd remove it and leave the previous 2